src/main/kotlin/com/openlattice/assembler/tasks/MaterializePermissionSyncTask.kt

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
This doesn't appear to be used 🤔

Reply:
It is used here: https://github.com/openlattice/conductor/pull/86/files